go/vt/vtgate/router_dml_test.go

```diff
@@ -477,6 +507,36 @@ func TestInsertGeneratorUnsharded(t *testing.T) {
 	}
 }
 
+func TestInsertAutoincUnsharded(t *testing.T) {
+	router, _, _, sbclookup := createRouterEnv()
+
+	// Fake a mysql auto-inc response.
+	query := "insert into simple(val) values ('val')"
+	wantResult := &sqltypes.Result{
+		Rows: [][]sqltypes.Value{{
+			sqltypes.MakeTrusted(sqltypes.Int64, []byte("1")),
+		}},
+		RowsAffected: 2,
```

Comment:
Done.
I wanted to use values that were not already present in the default result, but InsertID being 2 is sufficient for this test.